.github/workflows/ci.yml

```diff
@@ -152,6 +157,21 @@ jobs:
         shell: pwsh
         run: New-Item -ItemType SymbolicLink -Path "C:\${{ matrix.unity-version }}" -Target "C:\Program Files\Unity\Hub\Editor\${{ matrix.unity-version }}"
 
+      - name: Install Unity UPM Packages
```

Comment:
These steps are all called by Build but to get their timing more clearly, I'm adding them here explicitly here.
They only run if their output hasn't being built yet so the time spent on it will move from the "Build Unity" step to here